File: Model/Raca.cs
using System;

namespace Vaca.Model
{
    public class Raca
    {
        public Raca(string nom, double litres)
        {
            Nom = nom;
            LitresPerKg = litres;
        }

        public string Nom { get; }
        public double LitresPerKg { get; }

        public override string ToString()
        {
            return Nom;
        }
    }
}

File: src/main/java/net/xaviersala/Vaca/Model/Raca.java
/**
 * 
 */
package net.xaviersala.Vaca.Model;

/**
 * @author xavier
 *
 */
public class Raca {
	
	public static String desconeguda = "Desconeguda";
	
    private String Nom;    
    private double LitresPerKg;
	
    public Raca(String nom, double litres)
    {
    	if (nom == null || nom.trim().length() == 0 || nom.isEmpty()) { //JRE 1.8 fix
    		nom = desconeguda;
    	}
        Nom = nom;
        LitresPerKg = litres;
    }

    public String getNom() {
		return Nom;
	}
    
	public double getLitresPerKg() {
		return LitresPerKg;
	}

	@Override
	public String toString()
    {
        return Nom;
    }

}
